tests/test_foundry.py

```diff
@@ -116,11 +105,11 @@
     'n_items': 1000
 }
 
-# Globus endpoint for '_iris_dev'
-test_data_source = "https://app.globus.org/file-manager?origin_id=e38ee745-6d04-11e5-ba46-22000b92c6ec&origin_path=%2Ffoundry-test%2Firis-dev%2F"
+# Globus endpoint for '_iris_dev' for test publication
```

Comment:
@ascourtas , it seems like we're adding a new file here. Is this PR out of sync with the current testing strategy?

Reply:
hmm, I merged with def, that could be why

Reply:
@WardLT @Aadit-Ambadkar yes this is out of date -- currently in `dev` we have one test file (as it should be) with logic specifying which functions should be run in GHA and which are local-only. https://github.com/MLMI2-CSSI/foundry/blob/dev/tests/test_foundry.py 

my suggestion would be to pull the most recent changes to `dev` into this branch before merging into `dev`